env

```diff
@@ -0,0 +1,2 @@
+FIREBASE_RTD_BASE_URL=https://club-site-preview-default-rtdb.firebaseio.com/
+ENV_API_KEY=CDXzkjNFlZKhWLVZDZOBVmqCSHkM49FoGj
```

Comment:
This should not be included in our git history whatsoever!! Please, reset this commit